src/pages/home-page-slides/slide-details/slide-details.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useStyles } from './slide-details.styles';
+import HomePageSlideForm from '../../../components/home-page-slide-form';
+import { getAvailableSlides, getSlide } from '../../../redux/home-page-slides/home-page-slides.actions';
+import LoadingBar from '../../../components/loading-bar';
+
+const SlideDetails = ({ match }) =>{
+  const { id } = match.params;
+  const styles = useStyles();
+  const dispatch = useDispatch();
+  const { loading, slide,availableSlides } = useSelector(({ Slides }) => ({
+    loading: Slides.slideLoading,
+    slide: Slides.slide,
+    availableSlides: Slides.availableSlides
+  }));
+
+  useEffect(() => {
+    dispatch(getSlide(id));
+    dispatch(getAvailableSlides())
+  }, [dispatch]);
+
+  const slideOrder = availableSlides.filter(slideItem=>slideItem.show===true).length + 1
+
+  if (loading) {
+    return <LoadingBar />;
+  }
+  return(
+    <div className={styles.container}>
+      {slide !== null ? <HomePageSlideForm slide={slide} id={id} slideOrder={slideOrder} />: null}
```

Comment:
You can simplify this check here: `slide ?  <HomePageSlideForm  slide={slide} id={id} slideOrder={slideOrder} />: null`

Reply:
Fixed